### DISCUSSION TITLE
Importing + exporting masslynx files

### DISCUSSION THREAD

**Discussion by BobChallen** [michaelmarty/UniDec#78] [Q&A]
Hi! In using UniDec to deconvolute some MS/MS spectra, is it possible to import a masslynx .RAW file, deconvolute through UniDec, and then export again as a .RAW file? 

I'm imagining not as I presume UniDec alters the file structure, but holding out some hope!

Thanks

**Comment by michaelmarty** [comment]
Hi Bob,
Unfortunately, it's not possible. It's not an issue with structure but rather because I don't know of any libraries that can save back into Waters .Raw format. What are you hoping to do with the data? With some scripting, it would be possible to write back to .mzML if that would help. Thanks,
MTM
